coderd/util/time/duration.go

```diff
@@ -0,0 +1,12 @@
+package duration
+
+import (
+	"time"
+
+	"github.com/dustin/go-humanize"
+)
+
+func Humanize(d time.Duration) string {
+	endTime := time.Now().Add(d)
+	return humanize.Time(endTime)
```

Comment:
Given this is only used in two places, I'm not convinced we need any indirection in front of `humanize.Time`; we should just use it directly.

If we were using this very broadly, I could see some value is hiding this behind our own interface so we could swap out the underlying implementation, but I don't think we're there yet.

Reply:
Happy you noticed this to reduce abstraction!

Reply:
100%, I was just too lazy to remove the first abstraction I added. My bad 🙏